Rakefile

```diff
@@ -63,6 +61,9 @@ end
 
 namespace :gem do
   cross_platforms.each do |platform|
+    require 'rake_compiler_dock'
```

Comment:
Don't we require rake-compiler-dock at the top of this file as well?

Is it fair to say the entire Rakefile is a development-only concern? It's included in the gemspec (presumably because we need it for CI) but is it actually loaded by an end-user of the gem?

Reply:
> Is it fair to say the entire Rakefile is a development-only concern? It's included in the gemspec (presumably because we need it for CI) but is it actually loaded by an end-user of the gem?

Yeah, I think that's true. For some reason I thought the `Rakefile` might be needed when installing `--platform ruby`.